src/main/java/seedu/address/model/student/AssessmentStatistics.java

```diff
@@ -121,8 +121,7 @@ public double getMin() {
         Optional<Double> min = scores.stream()
                 .map(Score::getNumericValue)
                 .min(Comparator.naturalOrder());
-        assert min.isPresent();
-        return min.get();
+        return min.orElse(0.0);
```

Comment:
```suggestion
        return min.orElse(Score.MIN_VALUE);
```

Instead of using magic numbers here. Same for `getMax`, `getMedian` (and for `getMean`, which is currently not handling the case for no scores)